src/core/shape/frustum.js

```diff
@@ -135,7 +133,7 @@ class Frustum {
         let c = 0;
         for (let p = 0; p < 6; p++) {
             const plane = planes[p];
-            const d = plane[0] * scx + plane[1] * scy + plane[2] * scz + plane[3];
+            const d = plane.normal.x * scx + plane.normal.y * scy + plane.normal.z * scz + plane.distance;
```

Comment:
I'd probably do `const { normal, distance } = plane[p];` in a hope it might run faster. Likely no difference though.

Reply:
Done. Thanks!